flink-metrics/flink-metrics-datadog/src/main/java/org/apache/flink/metrics/datadog/DatadogHttpReporter.java

```diff
@@ -146,44 +146,38 @@ public void report() {
 	}
 
 	/**
-	 * Get config tags from config 'metrics.reporter.dghttp.tags'
-	 * */
+	 * Get config tags from config 'metrics.reporter.dghttp.tags'.
+	 */
 	private List<String> getTagsFromConfig(String str) {
 		return Arrays.asList(str.split(","));
 	}
 
 	/**
-	 * Get tags from MetricGroup#getAllVariables(), excluding 'host'
-	 * */
+	 * Get tags from MetricGroup#getAllVariables(), excluding 'host'.
+	 */
 	private List<String> getTagsFromMetricGroup(MetricGroup metricGroup) {
 		List<String> tags = new ArrayList<>();
 
 		for (Map.Entry<String, String> entry: metricGroup.getAllVariables().entrySet()) {
-			if(!entry.getKey().equals(HOST_VARIABLE)) {
+			if (!entry.getKey().equals(HOST_VARIABLE)) {
 				tags.add(getVariableName(entry.getKey()) + ":" + entry.getValue());
 			}
 		}
 
 		return tags;
 	}
 
-	/**
-	 * Get host from MetricGroup#getAllVariables() if it exists; returns Null otherwise
-	 * */
 	private String getHostFromMetricGroup(MetricGroup metricGroup) {
 		return metricGroup.getAllVariables().get(HOST_VARIABLE);
 	}
 
-	/**
-	 * Given "<xxx>", return "xxx"
```

Comment:
Could be helpful to leave a comment noting the removal of the leading and trailing angle brackets.